app/scripts/metamask-controller.js

```diff
@@ -1014,7 +1014,9 @@ export default class MetamaskController extends EventEmitter {
           this.metaMetricsController,
         ),
       },
-      undefined,
+      {
+        supportedChainIds: [CHAIN_IDS.MAINNET, CHAIN_IDS.GOERLI],
```

Comment:
Should Rinkeby be in this list as well?

Reply:
Rinkeby isn't supported by the swap api any more, right @dan437?

(And infura doesn't support it anymore either. And its removed from our custom network list.)